parm/config/config.base.nco.static

```diff
@@ -50,6 +50,7 @@ export DO_BUFRSND="YES"    # BUFR sounding products
 export DO_GEMPAK="YES"     # GEMPAK products
 export DO_AWIPS="YES"      # AWIPS products
 export WAFSF="YES"         # WAFS products
+export DO_VRFY="NO"        # VRFY step
```

Comment:
NCO doesn't run verification so this isn't needed. Can remove this line in config.base.nco.static.

Reply:
NCO does not run it. That is why default to NO here.